lib/tasks/package.js

```diff
@@ -25,6 +26,8 @@ class PackageTask extends Task {
     } else {
       // Assemble the project and pass to forge
       projectPath = quickTemp.makeOrRemake(this, '-tmpPath');
+      childProcess.execSync(`chmod a+x ${projectPath}`);
```

Comment:
Will this work on windows? Does it have a `chmod` command? I think I'd prefer to just move away from using `quickTemp` and write a tiny bit of logic to check/create this directory ourselves with the proper permissions from the outset...